src/main/java/cubicchunks/server/CubeProviderServer.java

```diff
@@ -403,6 +405,18 @@ private void populateCube(Cube cube) {
                 popcube.setPopulated(true);
             }
         });
+        
+        if (CubicChunks.Config.BoolOptions.USE_VANILLA_CHUNK_WORLD_GENERATORS.getValue()) {
```

Comment:
You may also want to check if it's not compatibility generator to avoid doing it twice. I also think it's not quite right as it doesn't generate cubes offset by 8 blocks. Does compatibility generator also not do that?